src/models/ratings/dto/ratings.dto.ts

```diff
@@ -0,0 +1,68 @@
+import { ApiProperty } from "@nestjs/swagger";
+export class Rating {
+	@ApiProperty()
+		id: number;
+	@ApiProperty()
+		name: string;
+	@ApiProperty()
+		rating: number;
+	@ApiProperty()
+		maxrating: number;
+	@ApiProperty()
+		ratedgames: number;
+	@ApiProperty()
+		isbot: boolean;
+}
+
+export class RatingList {
+	@ApiProperty()
+		items: Array<Rating>;
+	@ApiProperty()
+		total: number;
+	@ApiProperty()
+		perPage: number;
+	@ApiProperty()
+		page: number;
+	@ApiProperty()
+		totalPages: number;
+}
+
+export class RatingQuery {
+	@ApiProperty({
+		description: 'Number of results per page',
+		required: false
+	})
+		limit?: string;
```

Comment:
Should this be a `number`? Same with `RatingQuery .page`?

Reply:
Since these come from the query string they are strings and not numbers and need to be parsed after the fact.